packages/mco/components/create-dr-policy/selected-cluster-validator.tsx

```diff
@@ -0,0 +1,390 @@
+
+    React.useEffect(() => {
+      if (policyLoaded && !policyLoadError) {
+        dispatch({
+          type: DRPolicyActionType.SET_CLUSTER_SELECTION_VALIDATION,
+          payload: isSelectionValid,
+        });
+      }
+    }, [isSelectionValid, policyLoaded, policyLoadError, dispatch]);
+
+    const validations: ValidationType = validateClusterSelection(
+      selectedClusters,
+      drPolicies
+    );
+
+    const { peeringValidation, clusterValidation } = validations;
+
+    const invalidClusters: string[] = Array.from(
+      new Set([].concat(...Object.values(clusterValidation)))
+    );
+
+    const isInvalidPeering: boolean = Object.values(peeringValidation).some(
+      (validation) => validation
+    );
+
+    isSelectionValid = !isInvalidPeering && !invalidClusters.length;
+
+    // Warning
+    const isVersionMismatch =
+      selectedClusters[0].odfInfo?.odfVersion !==
+      selectedClusters[1].odfInfo?.odfVersion;
+
+    return (
+      <StatusBox
+        data={selectedClusters}
+        loaded={policyLoaded}
+        loadError={policyLoadError}
+        skeleton={
+          <StatusIconAndText
+            icon={<Spinner size="sm" />}
+            title={t(
+              'Running checks to ensure that the selected managed cluster meets ' +
+                'all necessary conditions so it can enroll in a Disaster Recovery policy.'
+            )}
+          />
+        }
+      >
+        {isSelectionValid ? (
+          <>
+            <Alert
+              data-test="odf-not-found-alert"
+              className="odf-alert mco-create-data-policy__alert"
+              title={t(
+                'All disaster recovery prerequisites met for both cluster.'
+              )}
+              variant={AlertVariant.success}
+              isInline
+            />
+            {isVersionMismatch && (
+              <Alert
+                data-test="odf-not-found-alert"
+                className="odf-alert mco-create-data-policy__alert"
+                title={t('Version mismatch across selected clusters')}
+                variant={AlertVariant.warning}
+                isInline
+              >
+                {t(
+                  'The selected clusters are running different versions of Data Foundation. ' +
+                    'Peering clusters with different versions can lead to potential issues and is not recommended. ' +
+                    'Ensure all clusters are upgraded to the same version before proceeding with peering to avoid operational risks.'
+                )}
+              </Alert>
+            )}
+          </>
+        ) : (
+          <>
+            {isInvalidPeering ? (
+              <PeeringValidationMessage peeringValidation={peeringValidation} />
+            ) : (
+              <>
+                <Alert
+                  data-test="odf-not-found-alert"
+                  className="odf-alert mco-create-data-policy__alert"
+                  title={t(
```

Comment:
provided the alter content says `The selected managed cluster does not`, the alert title should not use the singular also?
```suggestion
                    'The cluster does not meet the disaster recovery cluster prerequisites.'
```

Reply:
microcopy is still under discussion, this title and description will change,